HousingSearchApi/V1/Boundary/Requests/GetTenureListRequest.cs

```diff
@@ -6,5 +6,11 @@ public class GetTenureListRequest : HousingSearchRequest
     {
         [FromQuery(Name = "uprn")]
         public string Uprn { get; set; }
+
+        [FromQuery(Name = "bookingStatus")]
+        public string BookingStatus { get; set; }
+
+        [FromQuery(Name = "isTemporaryAccommodation")]
+        public string IsTemporaryAccommodation { get; set; }
```

Comment:
The way it's named, this should be `bool`.

What kind of values is this field dealing with?

Reply:
See Tuomo's comment below.